packages/protocol/thread/thread.ts

```diff
@@ -731,8 +731,6 @@ class _ThreadFront {
     } else if (rv.exception) {
       rv.exception = new ValueFront(pause, rv.exception);
     }
-    const { repaint } = await import("protocol/graphics");
-    repaint(true);
```

Comment:
Could you wrap this in a feature flag?

```js
if (app.prefs.repaintEvaluations) {
    const { repaint } = await import("protocol/graphics");
    repaint(true);
}
```

no need to expose the pref in settings, but nice to have

Reply:
sure!

Reply:
Done.